ext/NamedGraphsSimpleGraphAlgorithmsExt/NamedGraphsSimpleGraphAlgorithmsExt.jl

```diff
@@ -0,0 +1,17 @@
+module NamedGraphsSimpleGraphAlgorithmsExt
+using Graphs: AbstractGraph, edgetype
+using NamedGraphs: position_graph, vertices
+using NamedGraphs.GraphsExtensions: GraphsExtensions
+using SimpleGraphAlgorithms: SimpleGraphAlgorithms
+using SimpleGraphConverter: UG
```

Comment:
```suggestion
using SimpleGraphConverter: UndirectedGraph
```
I'd prefer to use the longer name so the code is more self-explanatory.